packages/eslint-config-airbnb-base/index.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     './rules/strict',
   ].map(require.resolve),
   parserOptions: {
-    ecmaVersion: 2018,
+    ecmaVersion: 'latest',
```

Comment:
this is a change we can't make unless babel-preset-airbnb is handling it by default. unfortunately i no longer maintain it, so I'm not familiar with what the latest version supports.

Reply:
seems it's no longer actively maintained - the last version was released ~4 years ago.
![image](https://github.com/airbnb/javascript/assets/13050025/abcc90e7-cc22-49e3-81ac-6c4a13f194de)

Reply:
accurate. but this is airbnb's eslint config, so my hands are kind of tied :-/